clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -720,6 +721,12 @@ private void maybeOverrideEnableAutoCommit(Map<String, Object> configs) {
         }
     }
 
+    private void checkGroupRemoteAssignor() {
+        if (getString(GROUP_PROTOCOL_CONFIG).equals(DEFAULT_GROUP_PROTOCOL) && getString(GROUP_REMOTE_ASSIGNOR_CONFIG) != null && !getString(GROUP_REMOTE_ASSIGNOR_CONFIG).isEmpty()) {
```

Comment:
if we do assert the "classic" protocol, we should replace `DEFAULT_GROUP_PROTOCOL` by `classic"

Reply:
Thanks for the review. Yeah, using `classic` directly make semantic more clear.